docker/BTC/docker-compose.yml

```diff
@@ -23,39 +23,86 @@ services:
   indexer:
     container_name: btc-indexer
     image: blockcore/indexer:0.2.32
+    deploy:
+        resources:
+            limits:
+              cpus: 0.200
+              memory: 256M
```

Comment:
Have you tried running the Bitcoin indexer with only 256BM RAM? How long did the indexing take then?

Reply:
Please can you recommend resource limits and I'll implement for all chains.

I have tested SBC, and will coordinate with Milad to test all the chains, but would like to use what we have here as a baseline to make the changes for all chains.

Reply:
Just set it to something for the different chains, and I'll test and verify one by one when it's all merged and I need to deploy it.

Reply:
Ok